README.md

```diff
@@ -284,6 +288,9 @@ non-italicized when typing the letters without typing a backslash first, like
 3.17 of the Short Math Guide][3.17]) plus some missing trig operators like
 `sech`, `arcsec`, `arsinh`, etc.
 
+You can also specify a speech-friendly representation of the operator name by supplying the operator name, a `|` and its speech alternative (separate multiple words with a `-`). For example, 'sin|sine cos|cosine tan|tangent sinh|hyperbolic-sine'.
```

Comment:
Agreed. The defaults looked kind of kludged together and I wasn’t keen on breaking something in this first round. @laughinghan your thoughts are welcome.